pkgs/by-name/er/errbot/package.nix

```
@@ -38,6 +38,7 @@ python3.pkgs.buildPythonApplication rec {
     slixmpp
     python-telegram-bot
     webtest
+    setuptools # for pkg_resources
```

```suggestion
    setuptools
    slixmpp
    python-telegram-bot
    webtest
```

not like it is undocumented https://github.com/errbotio/errbot/blob/653656d8dd96e84c3b7974bf2cbec30806bd6cdf/setup.py#L31